src/statsd_client.cpp

```diff
@@ -48,14 +75,14 @@ StatsdClient::StatsdClient(const string& host, int port, const string& ns)
     d = new _StatsdClientData;
     d->sock = -1;
     config(host, port, ns);
-    srandom(time(NULL));
+    srand(time(NULL));
```

Comment:
If you're going to change the random handling, might as well switch to C++11 and avoid polluting global state with srand().

```cpp
#include <random>

        std::random_device rd;
        std::default_random_engine engine { rd() };
        std::uniform_real_distribution<> dist { 0.0f, 1.0f };
        for(int i = 0; i < 10; ++i) std::cout << "Random between 0..1: " << dist(engine)  << std::endl;
```